pom.xml

```diff
@@ -1,341 +1,398 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Please revert back to spaces in this file.

Reply:
Ok